openapi/nytimes.newswire/types.bal

```diff
@@ -45,19 +45,19 @@ public type Article record {
 };
 
 public type InlineResponse200 record {
-    string copyright?;
-    int num_results?;
-    Article[] results?;
-    string status?;
+    string? copyright?;
```

Comment:
Fields are not originally nillable. Please regenerate without giving `--nullable` in the command.